pkg/featureflag/featureflag.go

```diff
@@ -26,6 +26,7 @@ import (
 // Remember to capture changes in the usage guide and release notes.
 var (
 	DynamicPlugins = new("DYNAMIC_PLUGINS", false)
+	CertRotation   = new("CERT_ROTATION", false)
```

Comment:
is Cert_rotation true in the deployment template? should this be consistent? or are there some security concerns.